src/api/Nhs.Appointments.Persistance/AvailabilityDocumentStore.cs

```diff
@@ -50,6 +51,14 @@ public async Task ApplyAvailabilityTemplate(string site, DateOnly date, Session[
         }
     }
 
+    public async Task<IEnumerable<DailyAvailability>> GetDailyAvailability(string site, DateOnly from, DateOnly to)
+    {
+        var docType = documentStore.GetDocumentType();
+        var documents = await documentStore.RunQueryAsync<DailyAvailabilityDocument>(b => b.DocumentType == docType && b.Site == site && b.Date >= from && b.Date <= to);
+
+        return mapper.Map<IEnumerable<DailyAvailability>>(documents);
```

Comment:
Since this is a typed document store, you don't need to invoke the mapper directly. If you've registered it in CosmosAutoMapperProfile you can just pass the target type to RunQueryAsync

Reply:
(I did the same thing in AvailabilityCreatedEVentDocumentStore if you want a copypasta)

Reply:
Ah TIL, will update.